ios-base/Makefile

```diff
@@ -0,0 +1,5 @@
+init:
+	xcodegen
+	bundle exec pod install
```

Comment:
The command `bundle install (--path vendor/bundle)` is missing? (or execute in manually?)

Reply:
Sorry, I missed. I will add `bundle install --path vender/bundle` 🙏.
Thank you for reporting! 😄

Reply:
Good catch. It would be nicer if `bundle install --path vendor/bundle` is also executed by this script.